runtime/Cargo.toml

```diff
@@ -16,6 +16,7 @@ serde = { features = ["derive"], optional = true, version = "1.0.136" }
 
 # encointer deps
 encointer-primitives = { default-features = false, git = "https://github.com/encointer/pallets", branch = "master" }
+encointer-balances-tx-payment = { default-features = false, git = "https://github.com/encointer/pallets", branch = "master" }
```

Comment:
Always also include the `std` feature when adding dependencies with `default-features` false.

It might have unforeseen consequences when this is not done. Substrate for instance infers a wasm environment sometimes.